packages/x-date-pickers/src/ClockPicker/ClockPicker.tsx

```diff
@@ -222,7 +221,7 @@ export const ClockPicker = React.forwardRef(function ClockPicker<TDate extends u
     components,
     componentsProps,
     date,
-    disableIgnoringDatePartForTimeValidation = false,
+    disableIgnoringDatePartForTimeValidation,
```

Comment:
Could be renamed to avoid double negation

```suggestion
    considerDatePartForTimeValidation,
```

Reply:
I don't understand this prop, we never set it to `true` and there is no issue on the core mentioning it
I'll leave it as is and try to know what was its initial goal